### DISCUSSION TITLE
future releases

### DISCUSSION THREAD

**Discussion by Rimmuru** [TechReborn/TechReborn#2920] [Q&A]
Will future releases fixing bugs specifically the release that fixed issue #2061 be backported to 1.18.1? 
or will 1.18.1 be discontinued now that you have a port for 1.18.2 out?

**Comment by Ayutac** [comment]
I think #2061 was fixed in 1.18.1 already?

**Reply by Rimmuru** [reply]
in the very last update on curse for 1.18.1 the issue is present unfortunately

**Reply by Ayutac** [reply]
does autocrafting table has two outputs or one in 1.18.1?

**Reply by Rimmuru** [reply]
it has 2. just a bit annoying as it eats the gobber2 staff instead of using the durability and unfortunately we have no other mod in the pack to do the same thing

**Reply by Ayutac** [reply]
wait, recipes with durability usage? The autocrafting table is not designed for that. I though from the issue that it was a normal recipe reminder! Then it's unlikely to work in 1.18.2-TR as well.

**Reply by Ayutac** [reply]
Tell me, is the transmutation stone a durability item as well?

**Comment by Ayutac** [comment]
Generally, no backports, see e.g. https://github.com/TechReborn/TechReborn/blob/4eccb0100e863c7edf614a9b6315b6a7fff886bb/.github/ISSUE_TEMPLATE.md (bit outdated, but the point is still up to date)
